ansible/roles/local_storage/templates/kafka-lv-brokers.yml.j2

```diff
@@ -0,0 +1,19 @@
+apiVersion: "local.storage.openshift.io/v1"
```

Comment:
Ditto on this file. (We should delete this file since it won't be in use anymore.)

Reply:
deleted